fdb-relational-server/src/main/java/com/apple/foundationdb/relational/server/FRL.java

```diff
@@ -88,6 +89,10 @@ public FRL() throws RelationalException {
     @SpotBugsSuppressWarnings(value = "CT_CONSTRUCTOR_THROW", justification = "Should consider refactoring but throwing exceptions for now")
     public FRL(@Nonnull Options options) throws RelationalException {
         final FDBDatabase fdbDb = FDBDatabaseFactory.instance().getDatabase();
+        final Long asyncToSyncTimeout = options.getOption(Options.Name.ASYNC_OPERATIONS_TIMEOUT_MILLIS);
```

Comment:
I suppose that given the fact that this is set at the DB level, we wouldn't be able to have that per query / per connection, is that correct?